### ISSUE TITLE
what about I change the architecture of the model?

### ISSUE THREAD

**Issue by ztt0821** [xinntao/BasicSR#26]
If I change the architecture of the model, can I use the pretrained model RRDB_PSNR_x4.pth?

**Comment by weiyuxingchen** [comment]
I have the same problem, how to get a pretrained model similar to RRDB_PSNR_x4.pth, if changing the model structure.

**Comment by xinntao** [comment]
Hi @ztt0821 
The pre-trained model RRDB_PSNR_x4.pth cannot be used if you change the model architecture. 

@weiyuxingchen 
If you change the network structure, you need to train a model by yourself instead of directly using the provided model. 
See [How to train - Train SR models](https://github.com/xinntao/BasicSR#usage) for more details.

**Comment by ztt0821** [comment]
Thank you for your answer. So the RRDB_PSNR_x4.pth is trained by the SR model?

**Comment by xinntao** [comment]
@ztt0821 
Yes,
RRDB means the architecture; PSNR means it is a PSNR-oriented model. 
It was trained by the SR model with L1 loss.

**Comment by ztt0821** [comment]
Ok. Got it. Thank you very much!
